pkg/playground/speed.js

```diff
@@ -226,6 +226,31 @@ function spawn() {
     });
 }
 
+function spawn_input() {
+    stop();
+
+    const el_result = document.getElementById("spawn-input-result");
+    el_result.textContent = "Running...";
+
+    start = Date.now();
+    total = 0;
+    channel = cockpit.spawn(["dd", "of=/tmp/spawninput"], { err: "message" });
+    const chunk = 'a'.repeat(65536);
+    for (let i = 0; i < 1000; i++) {
+        channel.input(chunk, true);
+        total += chunk.length;
+    }
+    channel.input(); // send EOF
+    channel
+            .then(() => {
+                el_result.textContent = "success";
+            })
+            .catch(ex => {
+                el_result.textContent = `failed with exit code ${ex.exit_status}: ${ex.message}`;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19325-20230921-183914-ded91df0-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)